doc/stdlib/math.rst

```diff
@@ -1684,6 +1684,12 @@ Statistics
 
    Compute the mean of whole array ``v``\ , or optionally along the dimensions in ``region``\ . Note: Julia does not ignore ``NaN`` values in the computation. For applications requiring the handling of missing data, the ``DataArray`` package is recommended.
 
+.. function:: mean(f::Function, v)
+
+   .. Docstring generated from Julia source
+
+   Apply the function `f` to each element of `v` and take the mean.
```

Comment:
I probably should have been able to figure out myself that "generated from Julia source" was auto-text and shouldn't be hand-written. 😕 I really appreciate your help and patience as I struggle through this. Still learning.